lib/breakpointhook.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function() {
```

Comment:
It'd be good to have a comment here explaining why this is awesome. Oh, and maybe it should go under debugger/